### DISCUSSION TITLE
Could not find PROTOC_GEN_C using the following names: protoc-gen-c

### DISCUSSION THREAD

**Discussion by sgavathe** [protobuf-c/protobuf-c#761] [Q&A]
I am seeing an error about the protoc-gen-c not found. Using conan package manager to install protobuf-c and its giving an error at 
find_package(protobuf-c REQUIRED CONFIG) of CMakeList.txt file.

```
-- Conan: Including build module from '/home/cnb/.conan2/p/b/proto04e4b1c778e7e/p/lib/cmake/protobuf/protobuf-options.cmake'
CMake Error at /home/cnb/.conan2/p/b/proto0e84357bacc50/p/lib/cmake/protobuf-c/protobuf-c.cmake:2 (find_program):
  Could not find PROTOC_GEN_C using the following names: protoc-gen-c
Call Stack (most recent call first):
  build/protobuf-c-config.cmake:37 (include)
  CMakeLists.txt:11 (find_package)
 ```

**Comment by bhutch29** [comment]
Did you ever determine a fix for this @sgavathe ? I am running into this myself today. Version `1.5.0`

**Comment by bhutch29** [comment]
It looks like the error comes from this line in the conan package's cmake file: https://github.com/conan-io/conan-center-index/blob/master/recipes/protobuf-c/all/protobuf-c.cmake#L2

**Comment by sgavathe** [comment]
@bhutch29 I have not been able to work on the fix for that error. Luckily, we only needed a support for protobuf-c++ using conan and we were able to establish that with no issues.

May be next month or so, when I have some time, I will get around this problem.

Meanwhile, if you happen to find any resolution , please let us know. 

Thank,
Shay
